docs/ARCHITECTURE.md

```diff
@@ -1,70 +1,33 @@
 
-It will be written in .NET Core 3.1.
+Additionally there is a 3rd party dependency with Leaflet for GIS maps and functionality.
 
-It will use a PostgreSQL database backend to store My Places that are created by users.
+![solution architecture](./images/arch-app-01.png)
 
-#### Endpoints
+### API Architecture
 
-| Endpoint                | Route                           | Params | Body    | Returns           | Note                                    |
-| ----------------------- | ------------------------------- | ------ | ------- | ----------------- | --------------------------------------- |
-| Get my places           | /api/my/places                  |        |         | Place[]           | [auth] filtered by current user         |
-| Get a place             | /api/my/places/{id}             | id     |         | Place             | [auth] only allow current users places  |
-| Add a place             | /api/my/places                  |        | Place   | Place             | [auth] link to current user             |
-| Update a place          | /api/my/places/{id}             | id     | Place   | Place             | [auth] only allow current user or admin |
-| Delete a place          | /api/my/places/{id}             | id     |         | Place             | [auth] only allow current user or admin |
-| Get all places          | /api/all/places                 |        | Place[] | [auth] only admin |
-| Get all places filtered | /api/all/places?userId={userId} | userId |         | Place[]           | [auth] only admin                       |
-| Get all users           | /api/users                      |        |         | User[]            | [auth]                                  |
+The back-end application is a .NET Core 3.1 RESTful API app. It is hosted in a linux container. It uses KeyCloak for authentication and authorization. Communication to it requires a JWT Bearer token.
 
-### Objects
+The API is connected to an MSSQL database hosted in a linux container.
 
-Currently the only objects uses are `Place` and `User`.
-Based on current functionality we should only need to capture `Place` in the database.
+![solution architecture](./images/arch-api-01.png)
 
-#### Place
+### OpenShift Architecture
 
-| Name        | Type     | Length | Required | Note     |
-| ----------- | -------- | ------ | -------- | -------- |
-| Id          | Int      |        | \*       | Identity |
-| Lat         | Float    |        | \*       |
-| Lng         | Float    |        | \*       |
-| Note        | String   | 2000   |
-| OwnerId     | Guid     |        | \*       |
-| CreatedOn   | DateTime |        | \*       |
-| UpdatedOn   | DateTime |
-| UpdatedById | Guid     |
+The DevOps platform that support CI/CD along with the Operations platform environments are hosted with OpenShift. Additionally KeyCloak which provides authentication and authorization is hosted within OpenShift.
```

Comment:
small typo: "The DevOps platform that support(s) ..."  <- _Note the missing (s)_

Reply:
fixed